api/PasswordCredential.json

```diff
@@ -1,156 +1,158 @@
-        "deprecated": false
-      }
-    },
-    "additionalData": {
+    "PasswordCredential": {
       "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/additionalData",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential",
         "support": {
           "chrome": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           },
           "webview_android": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           },
           "chrome_android": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           }
         },
         "status": {
           "experimental": true,
           "standard_track": true,
-          "deprecated": true
+          "deprecated": false
         }
-      }
-    },
-    "iconURL": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/iconURL",
-        "support": {
-          "chrome": {
-            "version_added": "52"
-          },
-          "webview_android": {
-            "version_added": "52"
+      },
+      "additionalData": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/additionalData",
+          "support": {
+            "chrome": {
+              "version_added": "51",
+              "version_removed": "60"
+            },
+            "webview_android": {
+              "version_added": "51",
+              "version_removed": "60"
+            },
+            "chrome_android": {
+              "version_added": "51",
+              "version_removed": "60"
+            }
           },
-          "chrome_android": {
-            "version_added": "52"
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": true
           }
-        },
-        "status": {
-          "experimental": true,
-          "standard_track": true,
-          "deprecated": false
         }
-      }
-    },
-    "idName": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/idName",
-        "support": {
-          "chrome": {
-            "version_added": "51",
-            "version_removed": "60"
-          },
-          "webview_android": {
-            "version_added": "51",
-            "version_removed": "60"
+      },
+      "iconURL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/iconURL",
+          "support": {
+            "chrome": {
+              "version_added": "52"
+            },
+            "webview_android": {
+              "version_added": "52"
+            },
+            "chrome_android": {
+              "version_added": "52"
```

Comment:
This is `false` all 3 flavors. I can't find any evidence that it was ever in Chrome. It would be `false` for both flavors of Opera as well.